src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -90,6 +90,36 @@ package object csv {
 
     /**
      * Saves DataFrame as csv files. By default uses ',' as delimiter, and includes header line.
+     * The resulting output will not be compressed.
+     */
+    def saveAsCsvFile(path: String) : Unit = {
+        saveAsCsvFile(path, Map(), null)
+    }
+
+    /**
+     * Saves DataFrame as csv files. By default uses ',' as delimiter, and includes header line.
+     * If the parameters Map contains the key "compressionCodec" with a value
+     * corresponding to the name of a class implementing
+     * org.apache.hadoop.io.compress.CompressionCodec then the output will be
+     * compressed.
+     */
+    def saveAsCsvFile(path: String, parameters: Map[String, String]): Unit = {
```

Comment:
I prefer not to add this method. ```saveAsCsfFile``` is a helper method for convenience. This functionality should be implemented in ``CsvParser```